utils/tf_solver.py

```diff
@@ -64,7 +64,7 @@ def set_objective(self, model_init_fn, dataset):
             ]
         else:
             self.lr_scheduler = lambda epoch: self.lr
-            self.wd_scheduler = lambda epoch: self.decoupled_wd
+            self.wd_scheduler = lambda epoch: self.decoupled_wd * self.lr
```

Comment:
Yep I don't know how this got through but indeed that's the correct way to scale the weight decay